### ISSUE TITLE
Error adding ANOData data to predictionIntercept in Richness model. 

### ISSUE THREAD

**Issue by RRTogunov** [gjearevoll/BioDivMapping#101]
https://github.com/gjearevoll/BioDivMapping/blob/dc5f65ce29269b60c1a8d02200988b44387d6708/pipeline/models/speciesModelRuns.R#L92-L93

Error adding ANOData data to predictionIntercept in Richness model. Presumably, this happens when a species group does not have any ANO data. 
```
>   workflow$modelOptions(INLA = list(num.threads = 12, control.inla=list(int.strategy = 'eb', cmin = 0),safe = TRUE),
+                         Richness = list(predictionIntercept = 'ANOData'))
Error in workflow$modelOptions(INLA = list(num.threads = 12, control.inla = list(int.strategy = "eb",  : 
  predictionIntercept needs to be a name of one of the datasets in the model.
```

We need a function that finds a relevant dataset for each species group.

**Comment by samaperrin** [comment]
Yep this will happen when there's no data in ANO for a given taxa/region. I don't think we can automate this, the dataset needs to be selected manually. Unless we want to create a list of suitable datasets by taxa and have a function that selects a dataset based on presence of suitable data and returns an error if there are none?

**Comment by Peprah94** [comment]
There is a problem with obtaining the ANOData for the entire Norway. Merging the SurveyPoint and the ANO_Art has no common Global ID. Therefore, it is impossible to get a dataset from ANOData in Norway, unless we can find another way of merging the two dataframe to obtain the ANOData required.
